README.md

```
@@ -33,19 +33,13 @@ development/               Local Podman Compose setup and configs
 
 ### To Build Container Image:
 
-_Linux/Windows_
-```shell
-export IMAGE=your-quay-repo
-make docker-build-push
-```
+Log in to quay.io first, then run the `docker-build-push` target with your image destination. The target will error with the appropriate `login` command if authentication is missing.
 
-_MacOS_
+    podman login quay.io   # or: docker login quay.io
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)